src/app/pages/reportsdashboard/reports-utils.worker.ts

```diff
@@ -53,7 +53,7 @@ function inferUnits(label: string) {
   return units;
 }
 
-function convertKMGT(input: number, units: string, fixed?: number) {
+function convertKMGT(input: number, units: string, fixed?: number): { value: number; prefix: string; shortName: string } {
```

Comment:
Could be a localized interface as it's being reused too

Reply:
Could be, but I am not too familiar with the code here to give it a proper name. Will do another time.